backend/src/main.py

```diff
@@ -1,25 +1,26 @@
 import asyncio
 import logging
-from hello_world.v1.greeter_rsm import Greeter
-from greeter_servicer import GreeterServicer
+from chat_servicer import ChatServicer
+from chat.v1.chat_rsm import Chat
 from resemble.aio.applications import Application
 from resemble.aio.workflows import Workflow
 
 logging.basicConfig(level=logging.INFO)
 
-EXAMPLE_GREETER_ID = 'greeter-hello-world'
+EXAMPLE_CHAT_ID = 'test'
 
 
 async def initialize(workflow: Workflow):
-    greeter = Greeter(EXAMPLE_GREETER_ID)
+    chat = Chat(EXAMPLE_CHAT_ID)
 
-    # Implicitly construct greeter upon first write.
-    await greeter.Greet(workflow, greeting="Hello, World!")
+    # Implicitly construct chat upon first write.
+    test = await chat.Create(workflow,)
```

Comment:
We don't need to set this equal to anything unless we're going to use it later.

Reply:
rodger that